Sources/Kitura/RouterRequest.swift

```diff
@@ -95,16 +95,15 @@ public class RouterRequest: SocketReader {
     //
     // Parsed Cookies, used to do a lazy parsing of the appropriate headers
     //
-    private var _cookies: Cookies?
+    private lazy var _cookies: Cookies = {
```

Comment:
This closure could probably use ```unowned self``` or ```weak self``` so there won't be a retain cycle. But I'm not sure retain cycle would happen at this point.